Makefile.am

```diff
@@ -62,14 +62,18 @@ endif
 
 EXTRA_DIST += contrib/configure-devel
 EXTRA_DIST += contrib/configure-release
+EXTRA_DIST += contrib/configure-release-mt
 EXTRA_DIST += contrib/configure-prof
 EXTRA_DIST += contrib/buildrpm.sh
 EXTRA_DIST += contrib/ucx_perftest_config/msg_pow2
 EXTRA_DIST += contrib/ucx_perftest_config/README
 EXTRA_DIST += contrib/ucx_perftest_config/test_types_uct
 EXTRA_DIST += contrib/ucx_perftest_config/test_types_ucp
 EXTRA_DIST += contrib/ucx_perftest_config/transports
-EXTRA_DIST += debian
+EXTRA_DIST += debian/changelog debian/compat debian/copyright debian/control \
```

Comment:
pls list each file in separate line, using `\`

Reply:
Fixed